ts/services/ReportService.ts

```diff
@@ -62,6 +62,30 @@ class ReportService {
     }
   }
 
+  public async ec2recommendation(config, logger): Promise<void> {
+    logger.debug('\nListing EC2RecommendationData');
+    try {
+      const reports = await this.getReports(config, logger);
+      // TODO
+      const EC2RecommendationReport = reports.filter( (report) => report.type === 'EC2Cost' );
```

Comment:
Check out https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find, it'll get you the first match or `undefined`. Then you don't need line 72.